schema_editor/app/scripts/resources/boundaries-service.js

```diff
@@ -0,0 +1,29 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Boundaries($resource, Config) {
+        var urlString = Config.api.hostname + '/api/boundaries/:id';
+
+        var resource = {
+          'query': stdResource.query,
+          'get': stdResource.get,
+          'post': postWithFile,
+          'remove': stdResource.remove,
+          'delete': stdResource.delete
+        };
+
+        var stdResource = $resource(urlString, {id: '@uuid'}, {
+            query: {
+                method: 'GET',
+                transformResponse: function(data) { return angular.fromJson(data).results; },
+                isArray: true
+            }
+        });
+
+        return resource;
```

Comment:
Yeah, this is a bit up in the air. I'm looking to maybe modify the file upload directive from OTI so that the data we want can just be patched on top of our uploaded shapefile using $resource. Angular's support for file uploading is a surprising blind spot in the framework